### PR TITLE
add schedTriggers and remove schedValue

### PR DESCRIPTION
**Pull request by QiufangMa**
schedValue is "The value which is written to the MIB object pointed to by
            schedVariable when the scheduler invokes an action."
schedVariable is "An object identifier pointing to a local MIB variable
            which resolves to an ASN.1 primitive type of INTEGER."
both seem parameters specific to SNMP, and thus may be omitted from the mapping?

Meanwhile, schedTriggers : "This variable counts the number of attempts (either
            successful or failed) to invoke the scheduled action." this seems to be related to leaf "counter" inside schedule-status grouping...

### CODE REVIEW

**Comment by boucadair** [comment]
I agree that schedValue is not applicable in our context. This is why we had it "not supported" in the table. Instead of deleting the entry, we may s/not supported/not applicable.

For schedTriggers, we do have both successful and failures in separate counters. Don't see the value to support this as a separate parameter. We may indicate in the table that it is covered by failure-counter + counter.

File: draft-ietf-netmod-schedule-yang.md [hunk 1]
```diff
@@ -469,7 +469,7 @@ can be mapped to YANG parameters.
 |schedLastFailure| Not Supported   |
 |schedLastFailed|   last-failed-occurrence|
 |schedStorageType|  Not Supported    |
-|schedValue|  Not Supported    |
+|schedTriggers|  counter    |
```
Comment (boucadair) [inline draft-ietf-netmod-schedule-yang.md:472]:
```suggestion
|schedValue|  Not Applicable|
|schedTriggers|  counter/failure-counter   |
```

**Comment by QiufangMa** [comment]
> I agree that schedValue is not applicable in our context. This is why we had it "not supported" in the table. Instead of deleting the entry, we may s/not supported/not applicable.
If this is the case, then i guess schedVariable should also be added in the table?

**Comment by boucadair** [comment]
> schedVariable

Agree, assuming we say not applicable.

**Review by boucadair — APPROVED** [review]
Thanks.

**Pull request merged**